src/main/java/land/oras/exception/ConfigLoadingException.java

```diff
@@ -0,0 +1,11 @@
+package land.oras.exception;
+
+public class ConfigLoadingException extends Exception {
```

Comment:
I would either use OrasException or extend OrasException

Reply:
okay, thanks better to orasexception itself for all kind of errors i will update it